### PR TITLE
Replace reactive store with Pinia

### PR DESCRIPTION
**Pull request by oceangravity**
Replace reactive store with Pinia

https://github.com/gioboa/vue-microfrontend-demo/issues/1

### CODE REVIEW

File: remote/vite.config.ts [hunk 1]
```diff
@@ -1,29 +1,10 @@
...
-      ? {
-          alias: {
-            vue: path.resolve(
-              __dirname,
-              "./node_modules/vue/dist/vue.runtime.esm-bundler.js"
-            ),
-            shared: path.resolve(__dirname, "../shared/shared"),
-          },
-        }
-      : {},
```
Comment (gioboa) [inline remote/vite.config.ts:26]: does dev mode work with this configuration?

Comment (oceangravity) [reply remote/vite.config.ts:26]: Nope 😯

Comment (gioboa) [reply remote/vite.config.ts:26]: I don't get why this PR remove that code.

Comment (oceangravity) [reply remote/vite.config.ts:26]: Sorry my bad, I deleted everything about the shared state 😥😥

Comment (gioboa) [reply remote/vite.config.ts:26]: Never mind 👍 I will close this PR and I will use your Pinia integration. I will mention you in the commit 😉 thanks for you help

**Pull request closed without merging**